pykechain/models/activity.py

```diff
@@ -428,11 +428,11 @@ def clone(
 
     def edit_cascade_down(
             self,
-            start_date: Optional[datetime.datetime] = None,
-            due_date: Optional[datetime.datetime] = None,
-            assignees: Optional[List[Text]] = None,
-            assignees_ids: Optional[List[Text]] = None,
-            status: Optional[Union[ActivityStatus, Text]] = None,
+            start_date: Optional[datetime.datetime] = Empty(),
```

Comment:
`Optional` is either a `datetime` or `None`. But you mean that `Optional` is either `datetime`, `None` or `Empty.

Maybe we have a create a new Type call `OptionalEmpty` or something in utils?

```suggestion
            start_date: Optional[datetime.datetime, Empty] = Empty(),
```

or 
```suggestion
            start_date: OptionalEmpty[datetime.datetime] = Empty(),
```

with
title: typing.py
```python3
OptionalEmpty = Union[None, Empty]

# Also defined in KE-chain backend so worthwhile to define here as well
ObjectID = Union[Text, UUID]
ObjectIDs = Union[Iterable[ObjectID], KeysView]
```

Reply:
Included the feedback